pydantic/_internal/_model_construction.py

```diff
@@ -384,6 +405,9 @@ def inspect_namespace(  # noqa C901
             and ann_type not in all_ignored_types
             and getattr(ann_type, '__module__', None) != 'functools'
         ):
+            if isinstance(ann_type, typing._AnnotatedAlias) and isinstance(ann_type.__metadata__[0], ModelPrivateAttr):
```

Comment:
I think you can use `pydantic._internal._typing_extra.is_annotated` to check if the annotation is `Annotated`, and check if `ModelPrivateAttr` is _in_ the metadata (not just the first one). Something like:

```suggestion
            if is_annotated(ann_type):
                _, *metadata = typing_extensions.get_args(ann_type)
                private_attr = next((v for v in metadata if isinstance(v, ModelPrivateAttr), None)
                if private_attr is not None:
                    ...
```

Reply:
Thanks I was trying around and looking for a functionality like this already existing

Reply:
Do we simply want to take the first `ModelPrivateAttr` or should we have an exception for multiple defined, as `Annotated` allows for * amount of metadata

Reply:
I'd say take the first one (which will using `next((...), None)`)